java/code/src/com/suse/manager/utils/test/PackageUtilsTest.java

```diff
@@ -19,23 +19,21 @@
 import com.redhat.rhn.domain.rhnpackage.PackageFactory;
 import com.redhat.rhn.domain.rhnpackage.test.PackageTest;
 import com.redhat.rhn.testing.BaseTestCaseWithUser;
-import com.suse.manager.utils.PackageUtils;
```

Comment:
This is not anymore the test for PackageUtils. Should we move these tests to PackageEvrTest?

Reply:
would make sense i did not do that the first time since i wanted to keep diff minimal but now we should do it.